suite-common/message-system/config/config.v1.json

```
@@ -1,8 +1,70 @@
 {
     "version": 1,
-    "timestamp": "2024-01-03T00:00:00+00:00",
+    "timestamp": "2025-01-06T00:00:00+00:00",
     "sequence": 74,
     "actions": [
+        {
+            "conditions": [
+                {
+                    "environment": {
+                        "desktop": ">=25.1.2",
+                        "mobile": ">=25.1.2",
+                        "web": ">=25.1.2"
+                    },
+                    "settings": [
+                        {
+                            "tor": true,
+                            "tbtc": true
+                        },
+                        {
+                            "tor": false,
+                            "tbtc": true
+                        }
```

I think there is no need to specify tor state.
```suggestion
                        {
                            "tbtc": true
                        }
```

---

And I believe the key is `test` not `tbtc`, but not sure if there are some changes.

---

Sure, thanks. Done https://github.com/trezor/trezor-suite/pull/16426/commits/46c797facaf0f7e4f2b1b24240fea2b63c7f04a8